instrumentation/opentelemetry-instrumentation-flask/pyproject.toml

```diff
@@ -38,7 +39,8 @@ instruments = [
 ]
 test = [
   "opentelemetry-instrumentation-flask[instruments]",
-  "markupsafe==2.0.1",
+  "flask >= 2.2.0",
```

Comment:
what would be a good way to test both pre-2.2.0 and post-2.2.0 flask in this repo?

Reply:
figured out tox.ini changes